<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RGBTempPC</class>
 <widget class="QMainWindow" name="RGBTempPC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RGBTempPC</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="temperatureInsideBox">
      <property name="title">
       <string>Temperature out</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="temperatureOutsideLabel">
         <property name="text">
          <string>--*C</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="temperatureOutsideBox">
      <property name="title">
       <string>Temperature in</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="temperatureInsideLabel">
         <property name="text">
          <string>-- *C</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="humidityBox">
      <property name="title">
       <string>Humidity</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="humidityLabel">
         <property name="text">
          <string>--%</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="modeBox">
      <property name="title">
       <string>Mode</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QRadioButton" name="temperatureInsideButton">
         <property name="text">
          <string>&amp;Temperature inside</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="temperatureOutsideButton">
         <property name="text">
          <string>Temperature outs&amp;ide</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="colorButton">
         <property name="text">
          <string>&amp;Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="lightButton">
         <property name="text">
          <string>&amp;Light</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="colorBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Color settings</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QSlider" name="colorSlider">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1499</number>
         </property>
         <property name="value">
          <number>500</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="lightBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Light settings</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QCheckBox" name="brightnessCheckBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Auto brightness</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="brightnessSlider">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>127</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="portSettingsBox">
      <property name="title">
       <string>Port settings</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="portNameLabel">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Port name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="portNamecomboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="connectButton">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
